components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,42 +176,40 @@ function renderIdeOption(
     version: IDEOption["imageVersion"],
     onSelect: () => void,
 ): JSX.Element {
-    const label = option.type === "desktop" ? "" : option.type;
+    const shouldShowOptionType = option.type !== "desktop" || option.title === "VS Code"; // Force show of "Desktop" in the list for VS Code Desktop
```

Comment:
taking a note on hardcoding, let's go with it for now

Reply:
Alternatively, we could add something like `forceShowType` in https://github.com/gitpod-io/gitpod/blob/main/install/installer/pkg/components/ide-service/ide_config_configmap.go to VS Code Desktop to remove this.